rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py

```diff
@@ -38,24 +38,31 @@ def getImageUploadAndColor():
     
     
     colorRequest = request.forms.get('color')
+    print(colorRequest)
     imageRequest = request.files.get('image')
     if not (colorRequest and imageRequest):
         abort(400, 'Bad request')
 
     if not colorRequest in colors:
         abort(407, 'Bad color input')
-
+    
     userHeightRequest = request.forms.get('userHeight')
-    if userHeightRequest is not None and not userHeightRequest.isnumeric():
-        abort(400, 'Bad user height number input')
-    else:
+    if userHeightRequest is None:
         userHeightRequest = ''
-
-    wallHeightRequest = request.forms.get('wallHeight')
-    if wallHeightRequest is not None and not wallHeightRequest.isnumeric():
-        abort(400, 'Bad wall height number input')
     else:
+        try:
+            user_HEIGHT = float(userHeightRequest)
```

Comment:
I assumed you checked this, but I figured I have to check, if you input a int does this break converting to float?

Reply:
Also don't put the variable with an underscore following height in all caps, follow the other naming conventions on this page

Reply:
Yes, I did check with casting to an int, it works fine. and okay I completely forgot about naming conventions lol, my b